apps/web/.env

```diff
@@ -47,6 +47,8 @@ MEET_JWT_APP_SECRET=
 # SENTRY
 SENTRY_ORG=ever-co
 SENTRY_PROJECT=ever-teams-web
+SENTRY_LOG_LEVEL=[info|debug]
```

Comment:
@NdekoCode is it possible to specific both values that way?? Or it should be one of them!?

Reply:
It should be one of them, Mr. @evereq, it's recommended during the build so that sentry can build the final configuration.